app/templates/base.html

```diff
@@ -49,6 +49,17 @@
                     </li>
                 </ul>
             </div>
+            <div class="language-toggle d-flex justify-content-end">
+                <form action="{% url 'set_language' %}" method="post" class="d-flex">
+                  {% csrf_token %}
+                  <input name="next" type="hidden" value="{{ redirect_to }}" />
+                  <select class="form-select form-select-sm me-2" name="language">
+                      <option value="en">English</option>
+                      <option value="af">Afrikaans</option>
```

Comment:
Please obtain this list from the configured languages. It should display alphabetically sorted. If it is correct in the settings, nothing is needed here, I guess.

Ideally we should preselect the currently selected language (possibly nothing), or otherwise the currently active language. The currently active language we get through something like this:
```
    {% get_current_language as LANGUAGE_CODE %}
```
I don't know of a way to get "nothing" if the user didn't yet select something, so we can go with the last option if needed.

Reply:
Okay, will do